src/gurt/dlog.c

```diff
@@ -172,8 +172,15 @@ static const char *clog_pristr(int pri)
 {
 	int s;
 
+#ifdef ORIG
 	pri = pri & DLOG_PRIMASK;
 	s = (pri >> DLOG_PRISHIFT) & 7;
+#else
+	s = DLOG_PRI(pri);
+	if ( s > (sizeof (norm) / sizeof (char *))){
```

Comment:
(style)  space prohibited between function name and open parenthesis '('
(style)  space prohibited between function name and open parenthesis '('
(style)  space required before the open brace '{'
(style)  space prohibited after that open parenthesis '('

Reply:
fixed